UPGRADE.md

```diff
@@ -34,6 +39,31 @@ As part of the update of flysystem the support for the guzzle client package `gu
 
 The `GoogleGeolocator` and the `NominatimGeolocator` no longer support the Guzzle client and require a `Symfony\HttpClient` client instead.
 
+### Updating to flysystem 3
+
+The Sulu media storage uses now the Flysystem Bundle which need to be configured:
```

Comment:
we should also mention if `superbalist/flysystem-google-storage` was used it need to be replaced by `league/flysystem-google-cloud-storage`.

And that the other flysystem adapters (s3, azure) and flysystem itself need update to Version 3 if used by the project.